src/shared.js

```diff
@@ -0,0 +1,50 @@
+const SIGN_INPUT_ERROR_CLASSNAME = "sign-input-error";
+const ERROR_MESSAGE_CLASSNAME = "error-message-on";
+const EMAIL_REGEX = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g;
+const PASSWORD_REGEX = /^[a-zA-Z0-9](?=.*[a-zA-Z])(?=.*[0-9]).{8,20}$/g;
+
+export const userID = {
+  email: "test@codeit.com",
+  password: "codeit101",
+};
+
+export function setInputError(elements, message) {
+  elements.input.className += ` ${SIGN_INPUT_ERROR_CLASSNAME}`;
+  elements.errorMessage.className += ` ${ERROR_MESSAGE_CLASSNAME}`;
+  elements.errorMessage.textContent = message;
+}
+
+export function removeInputError(elements) {
+  elements.input.classList.remove(SIGN_INPUT_ERROR_CLASSNAME);
+  elements.errorMessage.classList.remove(ERROR_MESSAGE_CLASSNAME);
+  elements.errorMessage.textContent = "";
+}
+
+export function isEmailValid(email) {
+  return new RegExp(EMAIL_REGEX).test(email);
+}
+
+export function isPasswordValid(password) {
+  return new RegExp(PASSWORD_REGEX).test(password);
+}
+
+export function togglePassword(input, toggleButton) {
+  if (input.getAttribute("type") === "password") {
+    input.setAttribute("type", "text");
+    toggleButton
+      .getElementsByTagName("img")[0]
+      .setAttribute("src", "./images/eye-on.svg");
+    return;
+  }
+  input.setAttribute("type", "password");
+  toggleButton
+    .getElementsByTagName("img")[0]
+    .setAttribute("src", "./images/eye-off.svg");
+}
+
+export function TokenStorage(Storage) {
```

Comment:
함수명과 argument이름을 PascalCase 작성해주신 이유가 있을까요~? 함수 이름도 동사로 시작하면 좋을 것 같아요 :)

```suggestion
export function TokenStorage(Storage) {
```